lib/scala/kg/src/main/scala/stores/kg/Neo4jKgStore.scala

```diff
@@ -371,11 +376,13 @@ final class Neo4jKgStore @Inject()(configuration: Neo4jStoreConfiguration) exten
       //          CREATE (:Node { id: node.id, label: node.label, aliases: node.aliases, pos: node.pos, datasource: node.datasource, other: node.other });
       transaction.run(
         """MATCH (subject:Node {id: $subject}), (object:Node {id: $object})
-          |CALL apoc.create.relationship(subject, $predicate, {datasource: $datasource, weight: toFloat($weight), other: $other}, object) YIELD rel
+          |CALL apoc.create.relationship(subject, $predicate, {datasource: $datasource, datasources: $datasources, id: $id, weight: toFloat($weight), other: $other}, object) YIELD rel
```

Comment:
You definitely don't need to restore redundant data.

Reply:
Removed in model changes